packages/components/src/VirtualizedList/RendererSelector.component.js

```diff
@@ -94,6 +99,14 @@ class RendererSelector extends React.Component {
 		} else {
 			ListRenderer = ListGrid;
 			customProps = { rowRenderer: getRowRenderer(type, this.props.rowRenderers) };
+
+			if (type === COLLAPSIBLE_PANEL) {
```

Comment:
Please avoid specific code in the RendererSeletor. If you need to pass options, perhaps you can :
- consider that the rowRenderer can hold an attribute `options`
- so 
```
ListRenderer = ListGrid;
const rowRenderer = getRowRenderer(type, this.props.rowRenderers);
customProps = { rowRenderer, ...rowRenderer.options };
```
- in RowCollapsiblePanel/index.js
```
const cache = new CellMeasurerCache({ fixedWidth: true });
RowCollapsiblePanel.options = {
	deferredMeasurementCache: cache,
	rowHeight: cache.rowHeight,
};
export default RowCollapsiblePanel;
```